src/components/lib/VToast/VToast.vue

```diff
@@ -0,0 +1,84 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+import { useStore } from 'vuex'
+
+import VStack from '@/components/lib/VStack/VStack.vue'
+import type { ToastMessage } from '@/store/toast.store'
+
+const store = useStore()
+
+const toastMessages = computed(() => store.getters.toastMessages)
+
+function handleCloseToast(toastMessage: ToastMessage) {
+  store.dispatch('closeToast', toastMessage)
+}
+</script>
+
+<template>
+  <div class="v-toast-container">
+    <div
+      v-for="toastMessage in toastMessages"
+      :key="`toast-message-${toastMessage.id}`"
+      class="v-toast"
+      :class="{
+        success: toastMessage.type === 'success',
+        error: toastMessage.type === 'error',
+        show: toastMessage.show,
```

Comment:
Used `TransitionGroup` instead of `Transition` as it fits our use-case
https://vuejs.org/guide/built-ins/transition-group.html

Resolved it here https://github.com/arcana-network/developer-dashboard/pull/98/commits/095227ffdb9fb9701afe1274203c968090cdb85b